site/content/3.12/release-notes/version-3.12/incompatible-changes-in-3-12.md

```diff
@@ -19,16 +19,32 @@ offer better resilience and synchronous replication. Also see the
 See [Single instance vs. Cluster deployments](../../deploy/single-instance-vs-cluster.md)
 for details about how a cluster deployment differs and how to migrate to it.
 
+## Pregel
+
+The distributed iterative graph processing (Pregel) system is no longer supported
+from v3.12 onward.
+
+In detail, the following functionalities have been removed:
+- All Pregel graph algorithms
+- The `PREGEL_RESULT()` AQL function
+- The `@arangodb/pregel` JavaScript API module
+- The Pregel HTTP API (`/_api/control_pregel/*`)
+- All `arangodb_pregel_*` metrics
+- The `pregel` log topic
+- The `--pregel.max-parallelism`, `--pregel.min-parallelism`, and
+  `--pregel.parallelism` startup options
+
 ## LDAP authentication
 
 Support for ArangoDB user authentication with an LDAP server in the
 Enterprise Edition has been removed.
 
 - All `--ldap.*` and `--ldap2.*` startup options have been removed
-- The `--server.authentication-timeout` startup option ...
-  TODO: still a regular option (not obsoleted) even though it previously stated "This option is only necessary if you use an external authentication system like LDAP."
 - The `--server.local-authentication` startup option has been obsoleted and
   will be fully removed in a future version
+- The `--server.authentication-timeout` startup option that mainly controlled
+  the caching for LDAP authentication now only controls the cluster-internal
+  authentication caching and shouldn't be touched
```

Comment:
not touched as in there should be no need for end-users to change it as it is only used internally as an optimization but when you change user accounts the cache is invalidated anyway so this could as well be hardcoded I suppose...